docs/source/oci/advanced/logging.rst

```diff
@@ -0,0 +1,49 @@
+.. _oci_logging:
+
+Logging and Output
+==================
+
+Writing Custodian Logs to Object Storage
+-----------------------------------------
+
+You can output logs and resource records to Object Storage.
+By default, Custodian will add the policy name and date as the prefix to the blob.:
+
+    .. code-block:: sh
+
+        custodian run -s oci://<Bucket_Name>/<Optional_Folder_Name>/ <policy file>
+
+In order to send data to a bucket in a different profile you can use query params
+
+    .. code-block:: sh
+
+        custodian run -s oci://<Bucket_Name>/<Optional_Folder_Name>/?profile=<NAME> <policy file>
+
+If the bucket does not exist then Custodian will try to create a bucket in the
+compartment identified by the environment variable "OCI_LOG_COMPARTMENT_ID".
+
+Writing Custodian Logs to OCI Logging Service
+---------------------------------------------
+
+You can send the logs to OCI Logging service. In order to use logging service,
+the environment variable "OCI_LOG_COMPARTMENT_ID" must be set with the value of
```

Comment:
rather then set log compartment id solely via env var, it seems like that could be done as well via url (host/domain, query param, etc)

Reply:
generally we want env var, cli flag, default.. for outputs cli flags as urls should allow encompassing all of the configuration for a given output, rather than requiring half the config in env vars, and half the configuration in cli param for a single output.

Reply:
Updated to enable query param as well, with query param taking precedence over env. variable. Still dont have a very good default value so there will be an error if the user fails to provide either of those